.coderabbit.yaml

```
@@ -0,0 +1,72 @@
+# CodeRabbit configuration file
+# This configures how CodeRabbit reviews your pull requests
+
+# Review settings
+reviews:
+  # Enable automatic reviews
+  auto_review: true
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)